plugins/defaultplugins/aclplugin/acl_config.go

```diff
@@ -291,14 +292,17 @@ func (plugin *ACLConfigurator) DeleteACL(acl *acl.AccessLists_Acl, callback func
 
 // DumpACL returns all configured ACLs in proto format
 // todo ACLDump/ACLDetails error invalid message ID 924, expected 922
-func (plugin *ACLConfigurator) DumpACL() (acls []*acl.AccessLists_Acl, err error) {
-	//acls, err := vppdump.DumpIPAcl(plugin.Log, plugin.vppChannel, measure.GetTimeLog(acl_api.ACLDump{}, plugin.Stopwatch))
-	//if err != nil {
-	//	plugin.Log.Error(err)
-	//	return nil
-	//}
-	//return acls
-	return  []*acl.AccessLists_Acl{},nil
+func (plugin *ACLConfigurator) DumpACL() ([]*acl.AccessLists_Acl, error) {
+	aclsWithIndex, err := vppdump.DumpIPAcl(plugin.Log, plugin.vppChannel, measure.GetTimeLog(acl_api.ACLDump{}, plugin.Stopwatch))
+	if err != nil {
+		plugin.Log.Error(err)
+		return nil, err
+	}
+	acls := make([]*acl.AccessLists_Acl, 0)
```

Comment:
It is unnecessary to initialize slice like this if you are just appending to it. It can be nil, like this:
```go
var acls []*acl.AccessLists_Acl
```

Reply:
goland code review barfs on this kind of slice initialization and requires you to do make(...). It started doing it in the last release. It was a surprise to me too. Not sure whether something changed in the Golang guidelines.

Reply:
I just tested and it *barfs* only for declaration like this:
```go
acls := []*acl.AccessLists_Acl{}
```
but this generates no error: